lms/djangoapps/teams/tests/test_views.py

```diff
@@ -662,6 +709,25 @@ def test_page_size(self):
         result = self.get_teams_list(200, {'page_size': 2})
         self.assertEqual(2, result['num_pages'])
 
+    def test_non_member_trying_to_get_private_topic(self):
```

Comment:
nit: possibly a more descriptive title?
```suggestion
    def test_non_member_cannot_access_private_topic(self):
```